src/tools/getBlock/getBlock.cpp

```diff
@@ -16,7 +16,7 @@
 //------------------------------------------------------------
 int main(int argc, const char * argv[]) {
 
-    etherlib_init();
+    etherlib_init(quickQuitHandler);
```

Comment:
This quitHandler should be specified at a more granular level

Reply:
Let's make the default behavior `quickQuitHandler` so end users will not have to specify this without getting the weird behavior of three-control+c to quit.

The easiest way to do this is like this:
1. Change the name of `defaultQuitHandler` to something more meaningful.
2. Leave `quickQuitHandler` as is
3. Remove the 'default' parameter to the function signature in the header and rebuild
4. The build will fail everywhere that doesn't have a handler specified
5. For all failures use the renamed (previously defaultHandler)
6. Get the build to work
8. Reenable default param in header
9. Remove use of quickQuit since it will now default
10. Rename quickQuit if you think it's worth it.